konan/konan.properties

```diff
@@ -25,8 +25,8 @@ dependencyProfiles = default alt
 airplaneMode = false
 # if true, ignores Xcode version check.
 ignoreXcodeVersionCheck = false
-# Disabled until decided better mechanism.
-# useFixedXcodeVersion = 11.0
+
+minimalXcodeVersion=11.0
```

Comment:
IIRC we agreed that there's a minimal version for building compiler and for using it.  Most important one is using part.

Reply:
This commit is right about check at the user side.